docs/smart_contract.md

```diff
@@ -0,0 +1,104 @@
+TODO: 
+1. Explain how to sync up the smart contract with the DKG
+2. Explain how to sync up the smart contract with the Shuffle
+3. Explain how security impacts the smart contract (at the decrypt stage)
+
+## Create an election
+This transaction requires the following 3 parameters:
```

Comment:
A detail, but it is more pleasant if we space the text with empty lines: After/before titles, code blocks, lists, and tables.